wifiphisher.py

```diff
@@ -875,10 +877,29 @@ def get_hostapd():
     # Only for systems with airmon-ng installed
     if os.path.isfile('/usr/sbin/airmon-ng'):
         proc = Popen(['airmon-ng', 'check', 'kill'], stdout=PIPE, stderr=DN)
-        
+
     # Get hostapd if needed
     get_hostapd()
 
+    # set the template path
+    TEMPLATE_PATH = phishingpage.get_path(TEMPLATE_NAME)
+
+    # if template exists in database but not locally ask for download
+    if not phishingpage.exists(TEMPLATE_PATH):
+        # get user's response
+        response = raw_input("Template is available online. Do you want to "\
+        "download it now? [Yy]")
```

Comment:
Use the y/n notation. Also you need a whitespace at the end. Check how it is done <a href="https://github.com/sophron/wifiphisher/blob/master/wifiphisher.py#L839">here</a>.